Source/WebCore/css/parser/CSSParserToken.cpp

```diff
@@ -486,15 +486,19 @@ CSSValueID CSSParserToken::id() const
 {
     if (m_type != IdentToken)
         return CSSValueInvalid;
-    if (m_id < 0)
-        m_id = cssValueKeywordID(value());
-    return static_cast<CSSValueID>(m_id);
+    return anyId();
 }
 
 CSSValueID CSSParserToken::functionId() const
 {
     if (m_type != FunctionToken)
         return CSSValueInvalid;
+    return anyId();
+}
+
+CSSValueID CSSParserToken::anyId() const
```

Comment:
Nit: I find the name `anyId` a little confusing. It sounds like it could return an arbitrary `CSSValueID`. What about `identOrFunctionId`?